docs/tools/terraform/getting-started.rst

```diff
@@ -0,0 +1,98 @@
+Set up your first Terraform project
+===================================
+
+This example shows the setup for a sample Terraform project and some useful commands to stand up (and destroy) your Aiven data infrastructure.
+
+Requirements 
+''''''''''''
+- `Download and install Terraform <https://www.terraform.io/downloads.html>`_
+- `Sign up <https://console.aiven.io/signup?utm_source=github&utm_medium=organic&utm_campaign=devportal&utm_content=repo>`_ for Aiven if you haven't already
+- `Generate an authentication token <https://developer.aiven.io/docs/platform/howto/create_authentication_token.html>`_ on Aiven's console or CLI
+
+
+1. ``provider.tf`` file:
```

Comment:
What action are we doing in this step? Let's explain the what and why, then show the code.